packages/common/src/relay/helpers.ts

```diff
@@ -45,12 +57,20 @@ export const preparePlayParams = (mediaList: (ICallingPlay | IRelayCallingPlay)[
       play.push({ type, params })
     }
   })
-  return play
+  return [play, volume]
+}
+
+export const preparePlayAudioParams = (params: string | { url: string, volume?: number }): [string, number] => {
+  if (typeof(params) === 'string') {
+    return [params, 0]
+  }
+  const { url, volume = 0 } = params
+  return [url, volume]
```

Comment:
@bryanrite now is good to go. Volume default to `0` if params is a string, instead use the one within the object.